.circleci/config.yml

```diff
@@ -168,7 +168,7 @@ jobs:
           docker_layer_caching: true
       - run: |
           GAIAD_VERSION=''
-          if [ "${CIRCLE_BRANCH}" = "master" ]; then
+          if [ "${CIRCLE_BRANCH}" = "main" ]; then
```

Comment:
not sure if this is correct @alessio

Reply:
I wouldn't bother too much. CircleCI integration should be replaced by GH actions. @marbar3778 thoughts?

Reply:
I thought it was already replaced?